_includes/specials/taproot/en/18-trivia.md

```diff
@@ -0,0 +1,83 @@
+{% auto_anchor %}
+
+- **What is a taproot?** Wikipedia [says][wikipedia taproot], "A taproot
+  is a large, central, and dominant root from which other roots sprout
+  laterally. Typically a taproot is somewhat straight and very thick, is
+  tapering in shape, and grows directly downward.  In some plants, such
+  as the carrot, the taproot is a storage organ so well developed that
+  it has been cultivated as a vegetable."
+
+    Bitcoin's [taproot][topic taproot], similar to the plant root, hides
+    the details of its scripts below the surface, away from public view,
+    and is very efficient at storing those scripts.
+
+- **Schnorr predates ECDSA:** we talk about [schnorr signatures][topic
+  schnorr signatures] as an upgrade on Bitcoin's original ECDSA
+  signatures because they make it easier to implement various
+  cryptographic tricks, but the schnorr signature algorithm predates the
+  DSA algorithm that ECDSA is based upon.  Indeed, DSA was created in
+  part to circumvent the [patent on schnorr signatures][schnorr patent]
+  and it was [reported][gov report] that "there were allegations of
+  infringement from Mr. Claus Schnorr and Public Key Partners (PKP)"
+  that DSA violated Schnorr's patent.  That patent expired in 2008.
```

Comment:
Just for my own curiosity: Everyone claims it's 2008 but google patents says 2010. I believe 2008 is right but is there a definitive source?

Reply:
So PAIR appears to be down, so I can't log in to check... but US patents receive an extension to their normal term due to delays in granting caused by the patent office, not too infrequently a year or two extra.  So that might be the origin of the 2010/2008 confusion. If so, then it wouldn't really be correct to say it expired in 2008.  Maybe best to just hedge and say 2008-2010. :)  I'd say it's irrelevant except 2010 would actually make it post Bitcoin's deployment (also might explain why EDDSA wasn't published until 2011), so it would be a point that was relevant to the story.

The matter is complicated by the fact that due to the drafting Schnorr's *US* patent wasn't *that* relevant:  Prior to  State Street Bank v. Signature (1998) (1) it was fairly unambiguous in the US that you couldn't patent an abstract algorithm, you had to patent a 'specific machine', and Schnorr's patent attorneys wrote the US patent in a way that would likely only apply to smartcards to avoid that problem (there were other hacks to evade that requirement without being very specific, e.g. saying the patent applied to a "computer readable medium", but apparently they weren't known to the drafter or they decided to be conservative and not use them).  In Europe where patenting abstract algorithms was less of an issue at the time, his patents were written more broadly and so it's actually his European patents that were considered more disruptive and presumably they had their own expiration times.

[(1) Subsequently, In Re Bilski overturned State Street in 2008, resulting in a whole bunch of US patents drafted between 1998 and 2008 having validity problems because they lacked any pretext of tying them to a machine... which has been its own adventure...]